cmd/webrtc-load-tester/roles/orchestrator.go

```diff
@@ -133,28 +135,39 @@ func initClients(cliFlags loadTestArguments) {
 	glog.Infof("Total number of viewers: %d\n", totalViewers)
 }
 
-func runLoadTest(ctx context.Context, args loadTestArguments) error {
-	stream, err := studioApi.CreateStream(api.CreateStreamReq{
-		Name: "webrtc-load-test-" + time.Now().UTC().Format(time.RFC3339),
-	})
-	if err != nil {
-		return fmt.Errorf("failed to create stream: %w", err)
+func runLoadTest(ctx context.Context, args loadTestArguments) (err error) {
+	var stream *api.Stream
+	if args.StreamID != "" {
+		stream, err = studioApi.GetStream(args.StreamID, false)
+		if err != nil {
+			return fmt.Errorf("failed to retrieve stream: %w", err)
+		}
+		glog.Infof("Retrieved stream with name: %s", stream.Name)
+	} else {
+		stream, err = studioApi.CreateStream(api.CreateStreamReq{
+			Name: "webrtc-load-test-" + time.Now().UTC().Format(time.RFC3339),
```

Comment:
I'd either remove `webrtc-` prefix at all or make the prefix based on protocol (hls/webrtc)